openedx/core/djangoapps/certificates/api.py

```diff
@@ -37,3 +38,14 @@ def _enabled_and_self_paced(course):
 
 def can_show_certificate_available_date_field(course):
     return _enabled_and_self_paced(course)
+
+
+def display_date_for_certificate(course, certificate):
+    if (
+        auto_certificate_generation_enabled_for_course(course)
+        and not course.self_paced
+        and course.certificate_available_date < datetime.now(UTC)
+    ):
```

Comment:
@sanfordstudent Can you move the ```and```s to the end? I _think_ that's the way.

Reply:
Yea I can do that fer ya.